config/helm/deployment-requests-limits.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: capa-controller-manager
+  namespace: capa-system
+spec:
+  template:
+    spec:
+      containers:
+        - name: manager
+          resources:
+            requests:
+              cpu: 100m
+              memory: 250Mi
+            limits:
+              cpu: 100m
+              memory: 250Mi
```

Comment:
I'm not sure about these numbers, feedback would be appreciate it. What I did is to use `kube-capacity` to see how much is actually using currently
```
NODE                                         NAMESPACE                 POD                                                                  CPU REQUESTS   CPU LIMITS     CPU UTIL      MEMORY REQUESTS   MEMORY LIMITS   MEMORY UTIL
ip-10-0-206-215.eu-west-2.compute.internal   giantswarm                capa-controller-manager-5c77c78dbc-t2ttc                             511m (12%)     0m (0%)        2m (0%)       284Mi (0%)        0Mi (0%)        163Mi (0%)
```